src/cbmc/cbmc_parse_options.cpp

```diff
@@ -450,6 +457,27 @@ void cbmc_parse_optionst::get_command_line_options(optionst &options)
   PARSE_OPTIONS_GOTO_TRACE(cmdline, options);
 }
 
+void cbmc_parse_optionst::check_smt2(
+  const std::string &name,
+  const std::string &pre_message)
+{
+  int exit_code;
+#ifdef _WIN32
+  {
+    std::string where_string = "%%windir%%\\system32\\where.exe";
+    exit_code = run(where_string, {where_string, name}, "", "", "");
+  }
+#else
+  exit_code = run("which", {"which", name}, "", "", "");
```

Comment:
This is a bit roundabout (maybe we should have a separate utility for querying the path ourselves?) but I suppose this is fine for now.